serve/mlc_serve/model/paged_cache_model.py

```diff
@@ -604,6 +604,9 @@ def __init__(
             config.sliding_window,
         )
 
+        if num_shards > 1:
+            model.disco_session.sync_worker_0()
```

Comment:
Synchronization here is necessary. Without it, `uvicorn` takes over the control before Disco is fully initialized, and that causes the model to get stuck at the first `tvm.nd.array(...)` with 100% GPU 0 utilization when it receives a request.